lib/datasource/rubygems/releases.ts

```diff
@@ -8,7 +9,10 @@ export function getReleases({
 }: GetReleasesConfig): Promise<ReleaseResult | null> {
   // prettier-ignore
   if (registryUrl.endsWith('rubygems.org')) { // lgtm [js/incomplete-url-substring-sanitization]
-      return getRubygemsOrgDependency(lookupName);
-    }
+    return getRubygemsOrgDependency(lookupName);
+  }
+  if (new URL(registryUrl).hostname === 'rubygems.pkg.github.com') {
```

Comment:
```suggestion
  if (parseUrl(registryUrl)?.hostname === 'rubygems.pkg.github.com') {
```

import `parseUrl` from `util/url`